core/chains/evm/headtracker/head_listener_test.go

```diff
@@ -117,7 +117,7 @@ func Test_HeadListener_NotReceivingHeads(t *testing.T) {
 	var chHeads chan<- *evmtypes.Head
 	var chErr = make(chan error)
 	var chSubErr <-chan error = chErr
-	sub := new(evmmocks.Subscription)
```

Comment:
Oh!  I must have missed this in my last PR.  This mock now gets generated in `core/chains/evm/client/mock` instead of `core/chains/evm/mock`.  We can remove the latter as it's just a static duplicate now (won't get regenerated any more)... and change this from `evmmocks` to `evmclimocks` (which I've found, GoLand somehow magically knows is a cue to adjust the import line to "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client/mocks"`!)

Reply:
Wait now I'm not sure I understand. Is there a conflict to resolve?

Reply:
Oh, nvm, I think I see now